### PR TITLE
feat: wip

### PR DESCRIPTION
**Pull request by Karlen9**

### CODE REVIEW

File: src/wash/common/CollectionNFTItem.tsx [hunk 1]
```diff
@@ -2,20 +2,29 @@
...
 import type { TNFTItem } from '../types/types';
+<<<<<<< Updated upstream
 import { titanOne } from './fonts';
 import { getPepeImage } from '../utils/getPepeImage';
+=======
+import { titanOne } from './WithFonts';
+import { Button } from './Button';
+import { useRouter } from 'next/navigation';
+import { InfoTooltip } from './InfoTooltip';
+>>>>>>> Stashed changes
```
Comment (Majorfi) [inline src/wash/common/CollectionNFTItem.tsx]: I guess merge conflict :D

File: src/wash/common/CollectionNFTItem.tsx [hunk 2]
```diff
@@ -136,24 +161,55 @@ export function CollectionNFTItem({ nft }: TNftItemProps): ReactElement {
...
       return (
+        <div style={{ position: 'relative' }}>
+          {nft?.isRare && (
+            <div
+              style={{
+                position: 'absolute',
+                top: '12px',
+                right: '12px',
+                zIndex: '1',
+              }}
```
Comment (Majorfi) [inline src/wash/common/CollectionNFTItem.tsx]: Let's move that to styled

File: src/wash/common/CollectionNFTItem.tsx [hunk 3]
```diff
@@ -136,24 +161,55 @@ export function CollectionNFTItem({ nft }: TNftItemProps): ReactElement {
   function renderNFTImage(): ReactNode {
     if (nft?.isRevealed) {
       return (
+        <div style={{ position: 'relative' }}>
```
Comment (Majorfi) [inline src/wash/common/CollectionNFTItem.tsx]: Let's move that to styled. You can do a `Relative` on the `theme.tsx` file maybe?

File: src/wash/common/CollectionNFTItem.tsx [hunk 4]
```diff
@@ -136,24 +161,55 @@ export function CollectionNFTItem({ nft }: TNftItemProps): ReactElement {
...
+              style={{
+                position: 'absolute',
+                top: '12px',
+                right: '12px',
+                zIndex: '1',
+              }}
+            >
+              <InfoTooltip
+                description={TOOLTIP_MESSAGES.goldenNft}
+                position={index % 4 < 2 ? 'right' : 'left'}
```
Comment (Majorfi) [inline src/wash/common/CollectionNFTItem.tsx]: Would this scale with mobile version?

File: src/wash/common/CurrentNFTBlock.tsx [hunk 5]
```diff
@@ -5,15 +5,19 @@ import { cl } from '../utils/utils';
...
 
 import { colors, mq, WashH2, type TColor } from '../utils/theme';
 import type { TItems, TNFTItem } from '../types/types';
 import type { TCleaningItem } from '../types/wash';
+<<<<<<< Updated upstream
 import { titanOne } from './fonts';
+=======
+import { titanOne } from './WithFonts';
+import { InfoTooltip } from './InfoTooltip';
+>>>>>>> Stashed changes
```
Comment (Majorfi) [inline src/wash/common/CurrentNFTBlock.tsx]: Merge error

File: src/wash/hooks/useReveal.tsx [hunk 6]
```diff
@@ -80,11 +83,17 @@ export function useReveal(refetchNft?: VoidFunction): TRevealHook {
           : 'An error occurred while revealing',
       );
     } finally {
-      await refetchNft?.();
+      Promise.all([refetchNft?.(), refetchCollection?.()]);
```
Comment (Majorfi) [inline src/wash/hooks/useReveal.tsx]:
```suggestion
      await Promise.all([refetchNft?.(), refetchCollection?.()]);
```

File: src/wash/utils/constants.ts [hunk 7]
```diff
@@ -178,6 +178,8 @@ export const TOOLTIP_MESSAGES = {
     'Every trade you make on Solana with Jumper helps wash your NFT clean. It takes $10,000 of volume to fully wash the NFT, or you can complete quests for power ups to boost your progress!',
   quest:
     'Every Solana swap you make on Jumper helps wash your NFT. But you can wash your NFT even faster by completing quests for items that boost your progress! Lfg.',
+  goldenNft:
+    'When the game ends on xx/xx/xxxx Golden NFT holders will be airdropped something sexy from the prize pool; holding Mad Lads, Retardio Cousins, Degods, Yoots, $Coby, and more!',
```
Comment (Majorfi) [inline src/wash/utils/constants.ts:253]:
```suggestion
    'When the game ends on xx/xx/xxxx Golden NFT holders will be airdropped something sexy from the prize pool; holding Mad Lads, Retardio Cousins, Degods, Yoots, $Coby, and more!', //TODO: change this on launch
```

**Review by Majorfi — CHANGES_REQUESTED** [review]

File: src/wash/common/CollectionNFTItem.tsx (hunk 4 above)
Comment (Karlen9) [reply src/wash/common/CollectionNFTItem.tsx]: Do we need tooltip on mobile version or we kill it?

**Pull request merged**